.github/workflows/java-windows.yml

```diff
@@ -16,10 +16,17 @@ jobs:
                 uses: actions/setup-java@v1
                 with:
                     java-version: ${{ matrix.java_version }}
+            -   name: check dexer
+                run: Invoke-WebRequest -Uri "https://github.com/Sable/android-platforms/blob/master/android-28/android.jar?raw=true" -OutFile "./android.jar"
```

Comment:
Let's try out if we can support API level 24 first, I think it should be enough for now. We can still increase to 26 in case we want to include java.nio.fie.Files

Btw. I liked the approach with wget, maybe curl works?
Don't forget to include a sha265sum check (sha256 for android-24 is `4c438bf1b3d4e6104faccc62e1ffe95ef197b88055497b540a871349e93c8685`
)

Reply:
Nah... let's increase to 26 already. I intend to get rid of the jdk8 extension in the long run and we have java.nio.file.Files in there.